test/resources/swagger.yml

```diff
@@ -76,6 +90,9 @@ paths:
           enum:
             - enum1
             - enum2
+          minLength: 1
+          maxLength: 2
+          maxItems: 3
```

Comment:
Do you have a ref to the swagger docs?

Reply:
https://swagger.io/docs/specification/data-models/data-types/#string
- minLength
- maxLength
https://swagger.io/docs/specification/data-models/data-types/#array-length
- maxItems